### ISSUE TITLE
SO_REUSEPORT has come up in production

### ISSUE THREAD

**Issue by danielballan** [NSLS-II/caproto#432]
From @bruceravel:

It looks like we urgently need solution to the issue with Python being compiled without the `SO_REUSEPORT` option.

```
(/opt/conda_envs/collection-2019-1.0-bmm) xf06bm@xf06bm-ws1 
[startup](master) > python -m caproto.ioc_examples.custom_write
[I 12:00:34.912 server:157] Asyncio server starting up...
[I 12:00:34.912 server:169] Listening on 0.0.0.0:5064
Traceback (most recent call last):
   File 
<snipped>
"/opt/conda_envs/collection-2019-1.0-bmm/lib/python3.6/asyncio/base_events.py", 
line 78, in _set_reuseport
     raise ValueError('reuse_port not supported by socket module')
ValueError: reuse_port not supported by socket module
```

**Comment by danielballan** [comment]
I'll have him give a try on the `trio` backend. Thanks for the link. I knew this was _discussed_, didn't realize it was fixed.

**Comment by klauer** [comment]
Closed by the somewhat hastily merged #220 (sorry 'bout that...)
